packages/graphql-js/tests/unit/documents.test.ts

```diff
@@ -0,0 +1,232 @@
+import { print } from 'graphql/language/printer'
+import { describe, expect, it } from 'vitest'
+import { toGraphQLDocument } from '../../src/graphql-object/query-builder'
+import schema from '../schemas/hasura'
+
+describe('main', () => {
```

Comment:
could you please add some tests to cover cases where multiple `where` arguments are used?

Reply:
done